lightning_server.cc

```diff
@@ -38,7 +81,21 @@ void LightningServer::start() {
 
   // Get the port to listen on
   std::vector<std::string> query = {"port"};
-  server_config_.propertyLookUp(query, port_);
+  bool found_port = server_config_.propertyLookUp(query, port_);
+  if (! found_port) {
+    std::cout << "No port specified in config, stopping server\n";
+    return;
+  }
+
+  // Set the number of threads if specified in the config
+  std::vector<std::string> query_path = {"numThreads"};
+  std::string num_threads;
+  bool found_threads = server_config_.propertyLookUp(query_path, num_threads);
+  num_threads_ = std::stoi(num_threads);
```

Comment:
Should this be the else-case of the if-statement below? The if-statement overrides this line right now, but it'd be clearer as: 

```c++
if (! found_threads) {
    // 0 default warning
} 
else {
    // Set num_threads_
}